.formatter.exs

```diff
@@ -0,0 +1,4 @@
+[
+  inputs: ["{mix,.formatter}.exs", "{core,eal,lib,local,test}/**/*.{ex,exs}"],
+  locals_without_parens: [defun: 2, defunp: 2]
```

Comment:
pls add [`defunpt/2`](https://hexdocs.pm/croma/Croma.Defun.html#defunpt/2) and [`defpt/2`](https://hexdocs.pm/croma/Croma.Defpt.html#defpt/2)

Reply:
Thank you. I fixed it.
1cace48cb5383ec8d22948b3331d87667c87f4fb

Currently, there are no functions defined with 'defunpt' or 'defpt' on a single line.
So this setting has no effect on the result of 'mix apply'.
However, 'defunpt' and 'defpt' are in frequent use and may one day be used to define a function on a single line.
For that reason, I decided beforehand to treat these functions as exceptions.